### PR TITLE
[BEAM-2083] Add Go SDK README

### PR DESCRIPTION
**Pull request by herohde**

### CODE REVIEW

**Comment by herohde** [comment]
R: @wcn3 @lostluck

File: sdks/go/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+The Go SDK is currently an experimental feature of Apache Beam and
+not suitable for production use. It is based on the following initial 
+[design](https://s.apache.org/beam-go-sdk-design-rfc).
+
+## How to run the examples
+
+**Prerequisites**: to use Google Cloud sources and sinks (default for
+most examples), follow the setup
+[here](https://beam.apache.org/documentation/runners/dataflow/). You can
+verify that it works by running the corresponding Java example, say.
```
Comment (lostluck) [inline sdks/go/README.md]: rm the colloquial ", say."

Comment (herohde) [inline sdks/go/README.md]: > **lostluck** wrote:
> rm the colloquial ", say."


Done.

**Comment by herohde** [comment]
R: @kennknowles (merge)

**Pull request merged**